src/build.py

```diff
@@ -965,6 +965,36 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsvu():
+  buildbot.Step('jsvu')
+
+  try:
+    if IsWindows():
+      # jsvu OS identifiers:
+      # https://github.com/GoogleChromeLabs/jsvu#supported-engines
+      os_id = 'windows64'
+      js_engines = 'chakra'
+    elif IsMac():
+      os_id = 'mac64'
+      js_engines = 'javascriptcore'
+    else:
+      return
+
+    # https://github.com/GoogleChromeLabs/jsvu#installation
+    proc.check_call(['npm', 'install', 'jsvu', '-g'])
```

Comment:
I'm guessing this isn't going to work as-is because npm won't be in the PATH. Take a look at how we do `NODE_BIN` up around line 200.

Reply:
Done.